src/families/tezos/bridge/js.ts

```diff
@@ -159,6 +147,19 @@ const getTransactionStatus = async (
     errors.amount = new NotEnoughBalance();
   }
 
+  // Catch a specific case that requires a minimum amount
+  if (
+    !errors.amount &&
+    t.mode === "send" &&
+    t.amount.lt(EXISTENTIAL_DEPOSIT) &&
+    (await api.getAccountByAddress(t.recipient)).type === "empty"
+  ) {
+    amountMustReset = true;
+    errors.amount = new NotEnoughBalanceBecauseDestinationNotCreated("", {
+      minimalAmount: "0.275 XTZ",
```

Comment:
I think there's an util to get the value from EXISTENTIAL_DEPOSIT + the currency's unit, right? But I can't find it...
Could be a small improvement instead of hardcoding it.

Reply:
definitely. and one other issue is that it's hardcoded but may change based on user's language.